assessment.md

```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
 ### Continuous Integration with Github Actions
 
 * Setup and configuration of GitHub Actions CI
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not use Github Actions CI or have it set up on their repository.
+  * Inadequate = Set up Github Actions to a workflow that doesn't make sense with the program and its contributors or in a way that isn't functionally committed to the repository.
+  * Adequate = Properly set up a Github Actions workflow in a way that is usable to collaborators.
+  * Good = Set up a Github Actions workflow that streamlines the workflow process and makes process consistent among devices that have different installations.
+  * Excellent = Streamlines all major angles including testing and general communication using Github Actions.
 
 * Performing secure releases of project deliverables
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not set up any security features on project deliverables.
+  * Inadequate = Sets up some security features on project deliverables, but project information is still easily compromisable.
+  * Adequate = Properly sets up all appropriate security features on project deliverables.
+  * Good = Properly sets up all appropriate security features on project deliverables in way that is well-suited for given project deliverables.
+  * Excellent = Properly sets up all appropriate security features on project deliverables in way that is best-suited for given project deliverables, and actively monitors project for potential security threats or improvements.
```

Comment:
The use of "security features" in this section is ambiguous. It would be more helpful if provided with examples of what can be counted as security(or insecure) features. I also encourage the team to do a little research on how to create a so-called "secure" release.